index.js

```diff
@@ -29,34 +29,57 @@ process.title = 'logdna-agent'
 commander._name = 'logdna-agent'
 commander
   .version(pkg.version, '-v, --version')
-  .description('This agent collect and ship logs for processing. Defaults to /var/log if run without parameters.')
-  .option('-c, --config <file>', `uses alternate config file (default: ${config.DEFAULT_CONF_FILE})`)
+  .description(
+    'This agent collect and ship logs for processing.'
+      + 'Defaults to /var/log if run without parameters.'
```

Comment:
Needs a space before `Defaults`

Reply:
should be `collects and ships logs`, no?

Reply:
Yeah, that makes more sense.